next-docs/pages/world/account-delegation.mdx

```diff
@@ -1 +1,74 @@
 # Account delegation
+
+_Account delegation_ allows a _delegator_ address to permit a _delegatee_ address to call a `System` on its behalf.
+This enables multiple use cases.
+
+- **Session wallets**.
+  Normally a wallet requires the user to authorize each transaction separately.
+  In the case of a blockchain game, this means having to authorize every move, which is excessive.
+  Using account delegation players can authorize a different wallet, one whose private key is stored on the client, to play to act on their behalf.
+  Because this wallet's private key is stored in the client, rather than a browser extension, the client can decide when asking for authorization is warranted and when it isn't.
+  By making sure this is a _separate_ wallet, we protect the player's main account in the case of vulnerable or malicious game clients.
+
+- **Approvals**.
+  MUD applications can have a feature similar to [the ERC-20 `approve` function](https://eips.ethereum.org/EIPS/eip-20#approve).
```

Comment:
Could we add some more details to this use case? Eg:

Users can approve contracts to perform actions in a MUD world on their behalf. This is conceptually similar to how `ERC20`s `approve`/`transferFrom` enables atopic swaps (by allowing contracts to withdraw from a user's balance and depositing a different asset in a single transaction), but works for any MUD system.